src/containers/App.js

```diff
@@ -1,13 +1,11 @@
 import React, { Component } from 'react';
 import { combineReducers } from 'redux';
 import { Provider } from 'react-redux';
-
 import { createStore, renderDevTools } from '../store_enhancers/devTools';
 
 import AppView from './AppView.js';
-import * as reducers from '../reducers';
+import reducer from '../reducers';
```

Comment:
Instead of using `combineReducers()` here, we'll do that in `reducers/index` so it's clearer what's going on.